core/src/main/java/cz/xtf/core/config/TestCaseContext.java

```diff
@@ -0,0 +1,43 @@
+package cz.xtf.core.config;
+
+import static cz.xtf.core.config.OpenShiftConfig.OPENSHIFT_NAMESPACE;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class TestCaseContext {
+    /**
+     * Used only in case when test cases are executed in parallel in different namespaces when:
+     * -Djunit.jupiter.execution.parallel.mode.classes.default=concurrent -Dxtf.openshift.unique.namespace.pertestcase=true
+     *
+     * Maps Thread name -> testcase name.
+     */
+    public static Map<String, String> threadToCurrentlyRunningTestCaseMap = new HashMap<>();
```

Comment:
I think that limitation for static OpenShift client cannot be avoided but I believe I've found a way to avoid other limitation with creating OpenShift client in non-testcase thread using `org.junit.jupiter.api.extension.TestInstancePostProcessor`. I'll change the code to use it.